examples/olsconfig.yaml

```diff
@@ -21,3 +21,9 @@ OLSConfig:
     library_log_level: warning
   default_provider: bam
   default_model: ibm/granite-13b-chat-v2
+
+Authentication:
```

Comment:
This is probably a discussion for the other place, but why the top level keys are not in snake case (like the rest of the config)?

Reply:
@onmete 
Honestly no reason, i can patch hat.
simply i saw that the existing top level keys started with Capital letters so i followed the same.

if you think i have to patch, then i will.

Reply:
@syedriko is there a reason behind that?

Reply:
That's how I found them. There was a discussion here https://github.com/openshift/lightspeed-service/pull/166#discussion_r1456459619 but tbh I didn't catch what was agreed on.

Reply:
Sure, will perform the Changes accordingly